src/components/pages/home/MainView/Module/Module.jsx

```diff
@@ -1,153 +1,105 @@
-              onChangeIndex={this.onChangeIndex(tiles.length)}
-              onSwitching={this.onSwitching}
-              onTransitionEnd={this.onTransitionEnd}
-              resistance
-              slideClassName="module-slides"
-            >
-              {tiles.map((tile, index) => {
-                return isOneItemLayout ? (
-                  <OneItem
-                    historyPush={historyPush}
-                    isSwitching={isSwitching}
-                    // eslint-disable-next-line react/no-array-index-key
-                    key={`${index}-tile`}
-                    layout={layout}
-                    moduleName={title}
-                    parsedParameters={parsedParameters}
-                    row={row}
-                    tile={tile}
-                  />
-                ) : (
-                  <TwoItems
-                    historyPush={historyPush}
-                    isSwitching={isSwitching}
-                    // eslint-disable-next-line react/no-array-index-key
-                    key={`${index}-tile`}
-                    layout={layout}
-                    moduleName={title}
-                    parsedParameters={parsedParameters}
-                    row={row}
-                    tile={tile}
-                  />
-                )
-              })}
-            </SwipeableViews>
-          </ul>
-        </section>
-      )
-    )
-  }
-}
-
-Module.defaultProps = {
-  geolocation: {
-    latitude: null,
-    longitude: null,
-  },
+  return (
+    <section className="module-wrapper">
+      <h1>
+        {title}
+      </h1>
+      <ul>
+        <SwipeableViews
+          className={layout || PANE_LAYOUT['ONE-ITEM-MEDIUM']}
+          disableLazyLoading
+          enableMouseEvents
+          hysteresis={swipeRatio}
+          onChangeIndex={onChangeIndex(tiles.length)}
+          onSwitching={onSwitching}
+          onTransitionEnd={onTransitionEnd}
+          resistance
+          slideClassName="module-slides"
+        >
+          {tiles.map((tile, index) => {
+            return isOneItemLayout ? (
+              <OneItem
+                historyPush={historyPush}
+                isSwitching={isSwitching}
+                // eslint-disable-next-line react/no-array-index-key
+                key={`${index}-tile`}
+                layout={layout}
+                moduleName={title}
+                parsedParameters={parsedParameters}
+                row={row}
+                tile={tile}
+              />
+            ) : (
+              <TwoItems
+                historyPush={historyPush}
+                isSwitching={isSwitching}
+                // eslint-disable-next-line react/no-array-index-key
+                key={`${index}-tile`}
+                layout={layout}
+                moduleName={title}
+                parsedParameters={parsedParameters}
+                row={row}
+                tile={tile}
+              />
+            )
+          })}
```

Comment:
```suggestion
          {tiles.map((tile, index) =>
              <LayoutComponent
                historyPush={historyPush}
                isSwitching={isSwitching}
                // eslint-disable-next-line react/no-array-index-key
                key={`${index}-tile`}
                layout={layout}
                moduleName={title}
                parsedParameters={parsedParameters}
                row={row}
                tile={tile}
              />
          )}
```
Si tu mets `const LayoutComponent = isOneItemLayout ? OneItem : TwoItems` plus haut
ça améliorerait la lisibilité ?